### ISSUE TITLE
[ScrollView] How does scroll respnder "steal" the touch lock of a touchable

### ISSUE THREAD

**Issue by Fanghao** [facebook/react-native#1903]
If I am pressing a touchableOpacity in a list view and then scroll vertically, the touchable event will be terminated.

So I am thinking doing something similar. Basically a panResponder to horizontally swipe to close a side panel and that side panel is a list view of touchableOpacities. But couldn't get the desired result like scrollResponder which "steals" the touch lock. So lots of time the panel is closed but the onPress is called as well. 

Any suggestion?

    this.panResponder = PanResponder.create({
      onMoveShouldSetPanResponder: this.handleMoveShouldSetPanResponder,
      onPanResponderMove: this.handlePanResponderMove,
      onPanResponderRelease: this.handlePanResponderEnd,
      onPanResponderTerminate: this.handlePanResponderEnd
    });

    handleMoveShouldSetPanResponder(evt, {vx, vy, numberActiveTouches}) {
      // steal from here?
      return numberActiveTouches == 1 && Math.abs(vx) > 0.2 && Math.abs(vy) < 0.2;
    }

**Comment by browniefed** [comment]
For more implementation type questions/suggestions we recommend asking on StackOverflow as you have a better chance of getting helped. Your question is  a little hard to follow, but you should disable scrolling on the scroll view with the prop `scrollEnabled`

@brentvatne

**Comment by brentvatne** [comment]
Yeah this is something that I would like to see fixed too - it would be great if we were able to block the native ScrollView from becoming responder. Dup of #1046 although worded slightly differently :) See my comment at the bottom for more info.
